tests/utils/test_sample_trajectory.py

```diff
@@ -210,16 +354,31 @@ def test_record_corrector(
         torch.tensor(sample_trajectory.data["corrector_sigma"]),
         torch.repeat_interleave(list_sigmas, number_of_corrector_steps),
     )
-    torch.testing.assert_close(
-        torch.stack(sample_trajectory.data["corrector_x_i"], dim=0), list_x_i_corr
-    )
-    torch.testing.assert_close(
-        torch.stack(sample_trajectory.data["corrector_corrected_x_i"], dim=0),
-        list_corrected_x_i,
-    )
-    torch.testing.assert_close(
-        torch.stack(sample_trajectory.data["corrector_scores"], dim=0), corrector_scores
-    )
+    for axl_field, axl_name in AXL_NAME_DICT.items():
+        corrector_i = torch.stack(
+            sample_trajectory.data[f"corrector_{axl_name}_i"], dim=0
+        )
+        target_corrector_i = torch.stack(
+            [getattr(axl, axl_field) for axl in list_axl_i_corr], dim=0
+        )
+        torch.testing.assert_close(corrector_i, target_corrector_i)
+        corrector_corrected_im1 = torch.stack(
```

Comment:
Shouldn't this be `corrector_corrected_im1` -> `  corrector_corrected_i`? Why `im1`?

Same thing for the variables below.

Reply:
good enough.